pkg/users/group-details.js

```diff
@@ -0,0 +1,213 @@
+
+    const actions = [
+        <DropdownItem key="rename-group"
+                      onClick={ev => { setKebabOpen(false); rename_group_dialog(group) }}>
+            {_("Rename")}
+        </DropdownItem>,
+        <DropdownItem key="delete-group"
+                      className={group.uid === 0 ? "" : "delete-resource-red"}
+                      onClick={() => { setKebabOpen(false); delete_group_dialog(groups.find(g => g.name == group)) }}>
+            {_("Delete")}
+        </DropdownItem>
+    ];
+
+    const kebab = (
+        <Dropdown toggle={<KebabToggle onToggle={setKebabOpen} />}
+                isPlain
+                isOpen={isKebabOpen}
+                position="right"
+                dropdownItems={actions} />
+    );
+    return kebab;
+};
+
+export function GroupDetails({ groups, accounts, group }) {
+    return (
+        <Page groupProps={{ sticky: 'top' }}
+              isBreadcrumbGrouped
+              id="group"
+              breadcrumb={
+                  <Breadcrumb>
+                      <BreadcrumbItem to="#/">{_("Groups")}</BreadcrumbItem>
+                      <BreadcrumbItem isActive>{group}</BreadcrumbItem>
+                  </Breadcrumb>}>
+            <PageSection>
+                <Card className="group-details" id="group-details">
+                    <CardHeader>
+                        <CardTitle id="group-title"><Text component={TextVariants.h2}>{group}</Text></CardTitle>
+                        {superuser.allowed && <CardActions>
+                            <GroupActions groups={groups} group={group} accounts={accounts} />
+                        </CardActions>}
+                    </CardHeader>
+                    <CardBody>
+                        <Form isHorizontal onSubmit={apply_modal_dialog}>
+                            <FormGroup fieldId="group-name" hasNoPaddingTop label={_("Name")}>
+                                <output id="group-name">{group}</output>
+                            </FormGroup>
+                            <GroupAccountsSelect accounts={accounts} group={group} groups={groups} />
+                        </Form>
+                    </CardBody>
+                </Card>
+            </PageSection>
+        </Page>
+    );
+}
+
+export const GroupAccountsSelect = ({ accounts, group, groups }) => {
+    const current_group = groups.find(g => g.name == group);
+    const [isOpenAccount, setIsOpenAccount] = useState(false);
+    const [selected, setSelected] = useState([...current_group.userlist, ...current_group.userlistPrimary]);
+    const [history, setHistory] = useState([]);
+    const [loggedInUsersAffected, setLoggedInUsersAffected] = useState(false);
+
+    const undoAccountChanges = () => {
+        const undoItem = history[history.length - 1];
+        if (undoItem.type === 'added') {
+            removeAccount(undoItem.name, true).then(() => setHistory(history.slice(0, -1)));
+        } else if (undoItem.type === 'removed') {
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230116-100807-e09cfe13-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)